文献を引用する記事が複数あるとコンパイルエラーになる

WOED 57 号は現段階で文献を引用する記事が 2 つあるが、`#bibliography` は 2 回以上使えないらしい。
無理に使おうとすると

```
error: multiple bibliographies are not yet supported
    ┌─ articles/typst-latex-comparison/main.typ:867:1
    │
867 │ #bibliography("ref.bib")
    │  ^^^^^^^^^^^^^^^^^^
```

でコンパイルエラーになる。

57 号の各記事に 
```typst
#import "@preview/alexandria:0.2.0": *

#show: alexandria(prefix: "", read: path => read(path))
```
を追加し、`#bibliography` を `#bibliographyx` に置換してコンパイルすると、コンパイルエラーになる。

<details>
<summary>エラーメッセージ（長い）</summary>

```
❯ make compile                                                         
typst compile --root=. --font-path=fonts main.typ main.pdf
downloading @preview/rubby:0.10.2
 15.8 KiB /  15.8 KiB (100 %)  15.8 KiB/s in 935.43 µs ETA: 0 s

warning: unknown font family: new computer modern sans
    ┌─ @preview/thmbox:0.2.0/src/thmbox.typ:146:26
    │
146 │           #set text(font: title-fonts, color, weight: "bold")
    │                           ^^^^^^^^^^^

warning: unknown font family: new computer modern sans
    ┌─ @preview/thmbox:0.2.0/src/thmbox.typ:167:26
    │
167 │           #set text(font: sans-fonts)
    │                           ^^^^^^^^^^

error: maximum function call depth exceeded
    ┌─ @preview/touying:0.6.1/src/utils.typ:165:2
    │
165 │   type(it) == content and it.func() == typst-builtin-sequence
    │   ^^^^^^^^

help: error occurred in this call of function `is-sequence`
   ┌─ @preview/cjk-unbreak:0.1.0/transform-childs.typ:35:7
   │
35 │     if utils.is-sequence(it) {
   │        ^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `transform-childs`
   ┌─ @preview/cjk-unbreak:0.1.0/remove-cjk-break-space.typ:27:9
   │
27 │   rest = transform-childs(rest, remove-cjk-break-space)
   │          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `remove-cjk-break-space`
   ┌─ @preview/cjk-unbreak:0.1.0/transform-childs.typ:46:21
   │
46 │       let new-body = transform-func(it.body)
   │                      ^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `transform-childs`
   ┌─ @preview/cjk-unbreak:0.1.0/remove-cjk-break-space.typ:27:9
   │
27 │   rest = transform-childs(rest, remove-cjk-break-space)
   │          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `remove-cjk-break-space`
   ┌─ @preview/cjk-unbreak:0.1.0/transform-childs.typ:43:21
   │
43 │       let new-body = transform-func(it.at("body", default: none))
   │                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `transform-childs`
   ┌─ @preview/cjk-unbreak:0.1.0/remove-cjk-break-space.typ:27:9
   │
27 │   rest = transform-childs(rest, remove-cjk-break-space)
   │          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `remove-cjk-break-space`
   ┌─ @preview/cjk-unbreak:0.1.0/transform-childs.typ:37:8
   │
37 │         transform-func(item)
   │         ^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `transform-childs`
   ┌─ @preview/cjk-unbreak:0.1.0/remove-cjk-break-space.typ:27:9
   │
27 │   rest = transform-childs(rest, remove-cjk-break-space)
   │          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `remove-cjk-break-space`
   ┌─ @preview/cjk-unbreak:0.1.0/transform-childs.typ:40:18
   │
40 │       let child = transform-func(it.child)
   │                   ^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `transform-childs`
   ┌─ @preview/cjk-unbreak:0.1.0/remove-cjk-break-space.typ:27:9
   │
27 │   rest = transform-childs(rest, remove-cjk-break-space)
   │          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `remove-cjk-break-space`
   ┌─ @preview/cjk-unbreak:0.1.0/transform-childs.typ:37:8
   │
37 │         transform-func(item)
   │         ^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `transform-childs`
   ┌─ @preview/cjk-unbreak:0.1.0/remove-cjk-break-space.typ:27:9
   │
27 │   rest = transform-childs(rest, remove-cjk-break-space)
   │          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `remove-cjk-break-space`
   ┌─ @preview/cjk-unbreak:0.1.0/transform-childs.typ:43:21
   │
43 │       let new-body = transform-func(it.at("body", default: none))
   │                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `transform-childs`
   ┌─ @preview/cjk-unbreak:0.1.0/remove-cjk-break-space.typ:27:9
   │
27 │   rest = transform-childs(rest, remove-cjk-break-space)
   │          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `remove-cjk-break-space`
   ┌─ @preview/cjk-unbreak:0.1.0/transform-childs.typ:37:8
   │
37 │         transform-func(item)
   │         ^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `transform-childs`
   ┌─ @preview/cjk-unbreak:0.1.0/remove-cjk-break-space.typ:27:9
   │
27 │   rest = transform-childs(rest, remove-cjk-break-space)
   │          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `remove-cjk-break-space`
   ┌─ @preview/cjk-unbreak:0.1.0/transform-childs.typ:43:21
   │
43 │       let new-body = transform-func(it.at("body", default: none))
   │                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `transform-childs`
   ┌─ @preview/cjk-unbreak:0.1.0/remove-cjk-break-space.typ:27:9
   │
27 │   rest = transform-childs(rest, remove-cjk-break-space)
   │          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `remove-cjk-break-space`
   ┌─ @preview/cjk-unbreak:0.1.0/transform-childs.typ:37:8
   │
37 │         transform-func(item)
   │         ^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `transform-childs`
   ┌─ @preview/cjk-unbreak:0.1.0/remove-cjk-break-space.typ:27:9
   │
27 │   rest = transform-childs(rest, remove-cjk-break-space)
   │          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `remove-cjk-break-space`
   ┌─ @preview/cjk-unbreak:0.1.0/transform-childs.typ:40:18
   │
40 │       let child = transform-func(it.child)
   │                   ^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `transform-childs`
   ┌─ @preview/cjk-unbreak:0.1.0/remove-cjk-break-space.typ:27:9
   │
27 │   rest = transform-childs(rest, remove-cjk-break-space)
   │          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `remove-cjk-break-space`
   ┌─ @preview/cjk-unbreak:0.1.0/transform-childs.typ:37:8
   │
37 │         transform-func(item)
   │         ^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `transform-childs`
   ┌─ @preview/cjk-unbreak:0.1.0/remove-cjk-break-space.typ:27:9
   │
27 │   rest = transform-childs(rest, remove-cjk-break-space)
   │          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `remove-cjk-break-space`
   ┌─ @preview/cjk-unbreak:0.1.0/transform-childs.typ:43:21
   │
43 │       let new-body = transform-func(it.at("body", default: none))
   │                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `transform-childs`
   ┌─ @preview/cjk-unbreak:0.1.0/remove-cjk-break-space.typ:27:9
   │
27 │   rest = transform-childs(rest, remove-cjk-break-space)
   │          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `remove-cjk-break-space`
   ┌─ @preview/cjk-unbreak:0.1.0/transform-childs.typ:37:8
   │
37 │         transform-func(item)
   │         ^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `transform-childs`
   ┌─ @preview/cjk-unbreak:0.1.0/remove-cjk-break-space.typ:27:9
   │
27 │   rest = transform-childs(rest, remove-cjk-break-space)
   │          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `remove-cjk-break-space`
   ┌─ @preview/cjk-unbreak:0.1.0/transform-childs.typ:40:18
   │
40 │       let child = transform-func(it.child)
   │                   ^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `transform-childs`
   ┌─ @preview/cjk-unbreak:0.1.0/remove-cjk-break-space.typ:27:9
   │
27 │   rest = transform-childs(rest, remove-cjk-break-space)
   │          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `remove-cjk-break-space`
   ┌─ @preview/cjk-unbreak:0.1.0/transform-childs.typ:37:8
   │
37 │         transform-func(item)
   │         ^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `transform-childs`
   ┌─ @preview/cjk-unbreak:0.1.0/remove-cjk-break-space.typ:27:9
   │
27 │   rest = transform-childs(rest, remove-cjk-break-space)
   │          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `remove-cjk-break-space`
   ┌─ @preview/cjk-unbreak:0.1.0/transform-childs.typ:40:18
   │
40 │       let child = transform-func(it.child)
   │                   ^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `transform-childs`
   ┌─ @preview/cjk-unbreak:0.1.0/remove-cjk-break-space.typ:27:9
   │
27 │   rest = transform-childs(rest, remove-cjk-break-space)
   │          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `remove-cjk-break-space`
   ┌─ @preview/cjk-unbreak:0.1.0/transform-childs.typ:37:8
   │
37 │         transform-func(item)
   │         ^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `transform-childs`
   ┌─ @preview/cjk-unbreak:0.1.0/remove-cjk-break-space.typ:27:9
   │
27 │   rest = transform-childs(rest, remove-cjk-break-space)
   │          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `remove-cjk-break-space`
   ┌─ @preview/cjk-unbreak:0.1.0/transform-childs.typ:40:18
   │
40 │       let child = transform-func(it.child)
   │                   ^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `transform-childs`
   ┌─ @preview/cjk-unbreak:0.1.0/remove-cjk-break-space.typ:27:9
   │
27 │   rest = transform-childs(rest, remove-cjk-break-space)
   │          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `remove-cjk-break-space`
   ┌─ @preview/cjk-unbreak:0.1.0/transform-childs.typ:37:8
   │
37 │         transform-func(item)
   │         ^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `transform-childs`
   ┌─ @preview/cjk-unbreak:0.1.0/remove-cjk-break-space.typ:27:9
   │
27 │   rest = transform-childs(rest, remove-cjk-break-space)
   │          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `remove-cjk-break-space`
   ┌─ @preview/cjk-unbreak:0.1.0/transform-childs.typ:40:18
   │
40 │       let child = transform-func(it.child)
   │                   ^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `transform-childs`
   ┌─ @preview/cjk-unbreak:0.1.0/remove-cjk-break-space.typ:27:9
   │
27 │   rest = transform-childs(rest, remove-cjk-break-space)
   │          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `remove-cjk-break-space`
   ┌─ @preview/cjk-unbreak:0.1.0/transform-childs.typ:37:8
   │
37 │         transform-func(item)
   │         ^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `transform-childs`
   ┌─ @preview/cjk-unbreak:0.1.0/remove-cjk-break-space.typ:27:9
   │
27 │   rest = transform-childs(rest, remove-cjk-break-space)
   │          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `remove-cjk-break-space`
   ┌─ @preview/cjk-unbreak:0.1.0/transform-childs.typ:40:18
   │
40 │       let child = transform-func(it.child)
   │                   ^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `transform-childs`
   ┌─ @preview/cjk-unbreak:0.1.0/remove-cjk-break-space.typ:27:9
   │
27 │   rest = transform-childs(rest, remove-cjk-break-space)
   │          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `remove-cjk-break-space`
   ┌─ @preview/cjk-unbreak:0.1.0/transform-childs.typ:37:8
   │
37 │         transform-func(item)
   │         ^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `transform-childs`
   ┌─ @preview/cjk-unbreak:0.1.0/remove-cjk-break-space.typ:27:9
   │
27 │   rest = transform-childs(rest, remove-cjk-break-space)
   │          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `remove-cjk-break-space`
   ┌─ @preview/cjk-unbreak:0.1.0/transform-childs.typ:40:18
   │
40 │       let child = transform-func(it.child)
   │                   ^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `transform-childs`
   ┌─ @preview/cjk-unbreak:0.1.0/remove-cjk-break-space.typ:27:9
   │
27 │   rest = transform-childs(rest, remove-cjk-break-space)
   │          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `remove-cjk-break-space`
   ┌─ @preview/cjk-unbreak:0.1.0/transform-childs.typ:37:8
   │
37 │         transform-func(item)
   │         ^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `transform-childs`
   ┌─ @preview/cjk-unbreak:0.1.0/remove-cjk-break-space.typ:27:9
   │
27 │   rest = transform-childs(rest, remove-cjk-break-space)
   │          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `remove-cjk-break-space`
   ┌─ @preview/cjk-unbreak:0.1.0/transform-childs.typ:40:18
   │
40 │       let child = transform-func(it.child)
   │                   ^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `transform-childs`
   ┌─ @preview/cjk-unbreak:0.1.0/remove-cjk-break-space.typ:27:9
   │
27 │   rest = transform-childs(rest, remove-cjk-break-space)
   │          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `remove-cjk-break-space`
   ┌─ @preview/cjk-unbreak:0.1.0/transform-childs.typ:37:8
   │
37 │         transform-func(item)
   │         ^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `transform-childs`
   ┌─ @preview/cjk-unbreak:0.1.0/remove-cjk-break-space.typ:27:9
   │
27 │   rest = transform-childs(rest, remove-cjk-break-space)
   │          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `remove-cjk-break-space`
   ┌─ @preview/cjk-unbreak:0.1.0/transform-childs.typ:40:18
   │
40 │       let child = transform-func(it.child)
   │                   ^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `transform-childs`
   ┌─ @preview/cjk-unbreak:0.1.0/remove-cjk-break-space.typ:27:9
   │
27 │   rest = transform-childs(rest, remove-cjk-break-space)
   │          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `remove-cjk-break-space`
   ┌─ @preview/cjk-unbreak:0.1.0/transform-childs.typ:37:8
   │
37 │         transform-func(item)
   │         ^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `transform-childs`
   ┌─ @preview/cjk-unbreak:0.1.0/remove-cjk-break-space.typ:27:9
   │
27 │   rest = transform-childs(rest, remove-cjk-break-space)
   │          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `remove-cjk-break-space`
   ┌─ @preview/cjk-unbreak:0.1.0/transform-childs.typ:37:8
   │
37 │         transform-func(item)
   │         ^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `transform-childs`
   ┌─ @preview/cjk-unbreak:0.1.0/remove-cjk-break-space.typ:27:9
   │
27 │   rest = transform-childs(rest, remove-cjk-break-space)
   │          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `remove-cjk-break-space`
   ┌─ @preview/cjk-unbreak:0.1.0/transform-childs.typ:40:18
   │
40 │       let child = transform-func(it.child)
   │                   ^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `transform-childs`
   ┌─ @preview/cjk-unbreak:0.1.0/remove-cjk-break-space.typ:27:9
   │
27 │   rest = transform-childs(rest, remove-cjk-break-space)
   │          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `remove-cjk-break-space`
   ┌─ @preview/cjk-unbreak:0.1.0/transform-childs.typ:37:8
   │
37 │         transform-func(item)
   │         ^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `transform-childs`
   ┌─ @preview/cjk-unbreak:0.1.0/remove-cjk-break-space.typ:27:9
   │
27 │   rest = transform-childs(rest, remove-cjk-break-space)
   │          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `remove-cjk-break-space`
   ┌─ @preview/cjk-unbreak:0.1.0/transform-childs.typ:40:18
   │
40 │       let child = transform-func(it.child)
   │                   ^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `transform-childs`
   ┌─ @preview/cjk-unbreak:0.1.0/remove-cjk-break-space.typ:27:9
   │
27 │   rest = transform-childs(rest, remove-cjk-break-space)
   │          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `remove-cjk-break-space`
   ┌─ @preview/cjk-unbreak:0.1.0/transform-childs.typ:37:8
   │
37 │         transform-func(item)
   │         ^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `transform-childs`
   ┌─ @preview/cjk-unbreak:0.1.0/remove-cjk-break-space.typ:27:9
   │
27 │   rest = transform-childs(rest, remove-cjk-break-space)
   │          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `remove-cjk-break-space`
   ┌─ @preview/cjk-unbreak:0.1.0/transform-childs.typ:40:18
   │
40 │       let child = transform-func(it.child)
   │                   ^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `transform-childs`
   ┌─ @preview/cjk-unbreak:0.1.0/remove-cjk-break-space.typ:27:9
   │
27 │   rest = transform-childs(rest, remove-cjk-break-space)
   │          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `remove-cjk-break-space`
   ┌─ @preview/cjk-unbreak:0.1.0/transform-childs.typ:37:8
   │
37 │         transform-func(item)
   │         ^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `transform-childs`
   ┌─ @preview/cjk-unbreak:0.1.0/remove-cjk-break-space.typ:27:9
   │
27 │   rest = transform-childs(rest, remove-cjk-break-space)
   │          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `remove-cjk-break-space`
   ┌─ @preview/cjk-unbreak:0.1.0/transform-childs.typ:37:8
   │
37 │         transform-func(item)
   │         ^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `transform-childs`
   ┌─ @preview/cjk-unbreak:0.1.0/remove-cjk-break-space.typ:27:9
   │
27 │   rest = transform-childs(rest, remove-cjk-break-space)
   │          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `remove-cjk-break-space`
   ┌─ @preview/cjk-unbreak:0.1.0/transform-childs.typ:40:18
   │
40 │       let child = transform-func(it.child)
   │                   ^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `transform-childs`
   ┌─ @preview/cjk-unbreak:0.1.0/remove-cjk-break-space.typ:27:9
   │
27 │   rest = transform-childs(rest, remove-cjk-break-space)
   │          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

make: *** [compile] Error 1
```

</details>

--- a/Makefile
+++ b/Makefile
@@ -1,13 +1,5 @@
 TYPST = typst
 
-.PHONY: check
-check:
-	./scripts/list.sh | ./scripts/check.sh
-
-.PHONY: setup
-setup:
-	./scripts/list.sh | ./scripts/install.sh
-
 .PHONY: compile
 compile:
 	$(TYPST) compile --root=. --font-path=fonts main.typ main.pdf
@@ -20,3 +12,11 @@ watch:
 clean:
 	rm -rf *.pdf
 	rm -rf fonts
+
+.PHONY: check
+check:
+	./scripts/list.sh | ./scripts/check.sh
+
+.PHONY: setup
+setup:
+	./scripts/list.sh | ./scripts/install.sh

--- a/Makefile
+++ b/Makefile
@@ -1,19 +1,22 @@
 TYPST = typst
 
-.PHONY: check setup compile watch clean
-
+.PHONY: check
 check:
 	./scripts/list.sh | ./scripts/check.sh
 
+.PHONY: setup
 setup:
 	./scripts/list.sh | ./scripts/install.sh
 
+.PHONY: compile
 compile:
 	$(TYPST) compile --root=. --font-path=fonts main.typ main.pdf
 
+.PHONY: watch
 watch:
 	$(TYPST) watch --root=. --font-path=fonts main.typ main.pdf
 
+.PHONY: clean
 clean:
 	rm -rf *.pdf
 	rm -rf fonts

--- a/articles/hinagata-markdown/main.typ
+++ b/articles/hinagata-markdown/main.typ
@@ -1,6 +1,6 @@
 #import "/template/article.typ": article
-#import "@preview/cmarker:0.1.7"
-#import "@preview/mitex:0.2.6": mitex
+#import "@preview/cmarker:0.1.8"
+#import "@preview/mitex:0.2.7": mitex
 
 #show: article.with(
   title: "記事を執筆しよう",

--- a/articles/hinagata-markdown/main.typ
+++ b/articles/hinagata-markdown/main.typ
@@ -18,17 +18,23 @@
   ),
   html: (
     pagebreak: ("void", _ => pagebreak()),
-    h: ("void", attrs => {
-      h(int(attrs.value) * 1pt)
-    }),
+    h: (
+      "void",
+      attrs => {
+        h(int(attrs.value) * 1pt)
+      },
+    ),
     v: ("void", attrs => v(int(attrs.value) * 1pt)),
-    img: ("void", attrs => figure(
-      image(
-        attrs.src,
-        width: eval(attrs.at("width", default: "auto")),
-        height: eval(attrs.at("height", default: "auto")),
+    img: (
+      "void",
+      attrs => figure(
+        image(
+          attrs.src,
+          width: eval(attrs.at("width", default: "auto")),
+          height: eval(attrs.at("height", default: "auto")),
+        ),
+        caption: attrs.alt,
       ),
-      caption: attrs.alt
-    ))
-  )
+    ),
+  ),
 )

--- a/template/init.typ
+++ b/template/init.typ
@@ -1,5 +1,5 @@
 #import "/template/utils.typ": body-setting, page-setting
-#import "@preview/cjk-unbreak:0.2.2": remove-cjk-break-space
+#import "@preview/cjk-unbreak:0.2.3": remove-cjk-break-space
 
 #let has-initialized-state = state("has-initialized-state", false)
 

--- a/template/init.typ
+++ b/template/init.typ
@@ -1,4 +1,4 @@
-#import "/template/utils.typ": page-setting, body-setting
+#import "/template/utils.typ": body-setting, page-setting
 #import "@preview/cjk-unbreak:0.2.2": remove-cjk-break-space
 
 #let has-initialized-state = state("has-initialized-state", false)
@@ -7,12 +7,10 @@
   show: page-setting
 
   context if not has-initialized-state.get() {
-
     show: remove-cjk-break-space
     show: body-setting
 
     body
-
   } else {
     body
   }

--- a/template/utils.typ
+++ b/template/utils.typ
@@ -150,12 +150,12 @@
   show raw: set text(font: fonts.raw)
   set raw(
     // コメントだけ色が薄いカラースキーム
-    theme: "assets/quiet.tmTheme"
+    theme: "assets/quiet.tmTheme",
   )
 
   show quote: set pad(0em)
   show quote.where(): set block(below: 0mm, above: 0mm)
-    show quote.where(): it => {
+  show quote.where(): it => {
     block(
       width: 100%,
       stroke: (left: (paint: luma(45%), thickness: 0.3mm)),
@@ -164,7 +164,7 @@
       above: 1.5em,
       below: 1.5em,
       breakable: false,
-      text(fill: luma(30%), it)
+      text(fill: luma(30%), it),
     )
   }
 
@@ -179,8 +179,7 @@
     grid(
       columns: (auto, 1fr),
       gutter: 0.75em,
-      numbering(sym.dagger + "1", ..counter(footnote).at(it.note.location())),
-      it.note.body,
+      numbering(sym.dagger + "1", ..counter(footnote).at(it.note.location())), it.note.body,
     )
   }
 

